update/release.go

```diff
@@ -196,7 +196,7 @@ func (s ReleaseSpec) calculateImageUpdates(rc ReleaseContext, candidates []*Serv
 
 	switch s.ImageSpec {
 	case ImageSpecLatest:
-		images, err = collectUpdateImages(rc.Registry(), candidates)
+		images, err = collectUpdateImages(rc.Registry(), candidates, logger)
```

Comment:
If the repo update fails then that image won't be included in the result and the failure will be logged. What alternative would you suggest?